src/context/ContextGetHelp.jsx

```diff
@@ -77,12 +77,8 @@ class ContextGetHelp extends React.Component {
             else if (this.state.selectedIndex > 0) {
                 e.preventDefault();
                 e.stopPropagation();
-
-                // the parent 'get help' option is not included in the getHelpOptions array
-                // but it is included as a selectedIndex, so there is an off by one that needs
-                // to be calculated, hence the -1
-                this.state.getHelpOptions[this.state.selectedIndex-1].onSelect();
-                return this.props.state;
+                this.props.closePortal();
```

Comment:
can't we call this.expand here instead of duplicating code?

Reply:
I'm not sure why, but if I use this.expand instead the app will crash after i expanded #disease status with this error 
![image](https://user-images.githubusercontent.com/15273981/60353762-b5760480-9998-11e9-90e3-34cdb3390e85.png)

And on other shortcut it fails to expanded and just return new lines